README.md

```diff
@@ -77,17 +73,20 @@ wav_gen_float = wav_gen.squeeze(0).cpu() # wav_gen is FloatTensor with shape [1,
 wav_gen_int16 = (wav_gen_float * 32767.0).numpy().astype('int16') # wav_gen is now np.ndarray with shape [1, T_time] and int16 dtype
 ```
 
-
 ## Local gradio demo <a href='https://github.com/gradio-app/gradio'><img src='https://img.shields.io/github/stars/gradio-app/gradio'></a>
+
 You can run a local gradio demo using below command:
+
 ```python
-pip install -r gradio_demo/requirements.txt
-python gradio_demo/app.py
+pip install -r demo/requirements.txt
+python demo/app.py
 ```
 
 ## Training
+
 Create symbolic link to the root of the dataset. The codebase uses filelist with the relative path from the dataset. Below are the example commands for LibriTTS dataset:
-``` shell
+
+```shell
 cd LibriTTS && \
```

Comment:
will need to cd into the new dir appropriately if we're to move the bundled LibriTTS filelists